### PR TITLE
[#537] Fix missing suffix

### PR DESCRIPTION
**Pull request by jhflorey**
This PR addresses comment https://github.com/episphere/connectApp/issues/537#issuecomment-1690340786 of issue https://github.com/episphere/connectApp/issues/537

### CODE REVIEW

File: siteManagerDashboard/participantSummary.js [hunk 1]
```diff
@@ -243,7 +243,9 @@ const renderDownload = async (participant, timeStamp, fileLocation, coordinates,
 const createSignature = (participant, type) => {
     const middleNameCID = type === 'hipaarevoc' ? fieldMapping.HIPAARevocMiddleName : type === 'datadestruction' ? fieldMapping.dataDestructionMiddleName : fieldMapping.consentMiddleName;
     const middleName =  participant[middleNameCID] ? ` ${participant[middleNameCID]} ` : ' ';
-    const createParticipantSignature = participant[fieldMapping.consentFirstName] + middleName + participant[fieldMapping.consentLastName]
+    const suffixCID = type === 'hipaarevoc' ? fieldMapping.HIPAARevocSuffix : type === 'datadestruction' ? fieldMapping.dataDestructionSuffix : fieldMapping.consentSuffix;
+    const suffix = participant[suffixCID] ? ` ${fieldMapping[participant[suffixCID]]}` : ''
+    const createParticipantSignature = participant[fieldMapping.consentFirstName] + middleName + participant[fieldMapping.consentLastName] + suffix
```
Comment (JoeArmani) [inline siteManagerDashboard/participantSummary.js:248]: can you wrap this with .trim() for cases where there's no suffix?

**Review by JoeArmani — APPROVED** [review]
looks good, one small comment.

File: siteManagerDashboard/participantSummary.js (hunk 1 above)
Comment (JoeArmani) [reply siteManagerDashboard/participantSummary.js:248]: My mistake, there's no trailing space. Please disregard.

Comment (we-ai) [reply siteManagerDashboard/participantSummary.js:248]: `participantSignature` as variable name is better than `createParticipantSignature` for a variable holding a string.

**Review by we-ai — APPROVED** [review]

**Pull request merged**